javascript/ql/src/Statements/UseOfReturnlessFunction.ql

```diff
@@ -67,8 +63,11 @@ predicate benignContext(Expr e) {
   // Avoid double reporting with js/property-access-on-non-object
   exists(PropAccess ac | ac.getBase() = e)
   or
-  // Avoid double-reporting with unused local.
+  // Avoid double-reporting with js/unused-local-variable
   exists(VariableDeclarator v | v.getInit() = e and v.getBindingPattern().getVariable() instanceof UnusedLocal)
+  or
+  // Avoid double reporting with js/call-to-non-callable
+  exists(InvokeExpr invoke | invoke.getCallee() = e)
```

Comment:
FYI. This can be written slighly shorter:

```
  any(InvokeExpr invoke).getCallee() = e
```

But it is rarely something we go back and do after a PR has been merged.